components/Hero.tsx

```diff
@@ -11,13 +11,24 @@ const { main: mainLanguages, others: otherLanguages } = languages;
 
 function Hero() {
   const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [searchEmpty, setSearchEmpty] = useState<boolean>(false);
```

Comment:
You don't need to pass in boolean in the generic because the default value is `false` i.e., boolean is inferred

Reply:
yes, of course, thank you for your comment , I will review it.